app/presenters/concerns/content_managed_view.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rich_text_renderer'
+
+module ContentManagedView
+  extend ActiveSupport::Concern
+  include ContentfulHelper
+
+  protected
+
+  def contentful_richtext_to_html(rich_text)
+    renderer = RichTextRenderer::Renderer.new
```

Comment:
It's probably worth memoizing this just in case additional rich text fields need to be rendered in future:
```ruby
@rich_text_renderer ||= RichTextRenderer::Renderer.new
```